extensions/kamelet/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,20 @@
+=== Pre-load Kamelets at build-time
+
+This extension allow to pre-load a set of Kamelets at build time through the following option. The kamelet to pre-load can be listed using the `quarkus.camel.kamelet.identifiers` property.
+
+=== Using the Kamelet Catalog
+
+//TODO: does not yet exist
```

Comment:
What does not exist? If there is something left to do, please file a follow up issue and paste the link here.

Reply:
this was a leftover.